backend/src/routes/v2/schema/getSchemaMigration.ts

```diff
@@ -0,0 +1,55 @@
+import bodyParser from 'body-parser'
+import { Request, Response } from 'express'
+import { z } from 'zod'
+
+import { AuditInfo } from '../../../connectors/audit/Base.js'
+import audit from '../../../connectors/audit/index.js'
+import { SchemaMigrationInterface } from '../../../models/SchemaMigration.js'
+import { searchSchemaMigrations } from '../../../services/schemaMigration.js'
+import { registerPath, schemaMigrationInterfaceSchema } from '../../../services/specification.js'
+import { parse } from '../../../utils/validate.js'
+
+export const getSchemaMigrationsSchema = z.object({
+  params: z.object({
+    name: z.string().optional().openapi({ example: 'My Schema Migration' }),
+  }),
+})
+
+registerPath({
+  method: 'get',
+  path: '/api/v2/schema-migrations',
```

Comment:
```suggestion
  path: '/api/v2/schema-migration',
```
Filename is singular, and `POST` endpoint is also singular.